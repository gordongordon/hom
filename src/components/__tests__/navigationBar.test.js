// Link.react-test.js
import React from 'react';
import NavigationBar from 'navigationBar.js';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';

test('From display', () => {
  const component = renderer.create(
<NavigationBar />     
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Check components', () => {

  const navigationBar = shallow(
    <NavigationBar/>
  );

  expect( navigationBar ).toHaveLength(1);
});