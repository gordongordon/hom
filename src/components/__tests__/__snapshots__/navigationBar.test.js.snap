// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`From display 1`] = `
<div>
  <div
    className="am-navbar am-navbar-light"
  >
    <div
      className="am-navbar-left"
      onClick={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="am-navbar-left-icon"
      >
        <svg
          className="am-icon am-icon-false am-icon-md"
          style={undefined}
        >
          <use
            xlinkHref="#false"
          />
        </svg>
      </span>
      <span
        className="am-navbar-left-content"
      />
    </div>
    <div
      className="am-navbar-title"
    >
      none
      ,
      HoMatching
    </div>
    <div
      className="am-navbar-right"
    >
      <div>
        <div
          onBlur={undefined}
          onClick={[Function]}
          onFocus={undefined}
          onMouseDown={[Function]}
          onMouseEnter={undefined}
          onMouseLeave={undefined}
          onTouchStart={[Function]}
          style={
            Object {
              "alignItems": "center",
              "display": "flex",
              "height": "100%",
              "marginRight": "-0.3rem",
              "padding": "0 0.3rem",
            }
          }
        >
          <span>
            請登入
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
